custom-distributed-tracing/README.md

```diff
@@ -0,0 +1,19 @@
+# New Relic .NET Agent Samples - Custom Distributed Tracing
+
+This folder contains a pair of sample applications to demonstrate how to use the .NET Agent's [distributed tracing APIs](https://docs.newrelic.com/docs/apm/agents/net-agent/configuration/distributed-tracing-net-agent/#manual-instrumentation) to link two instrumented services that communicate across a channel not automatically instrumented by the agent.
+The channel used for demonstration purposes is [Azure Service Bus](https://learn.microsoft.com/en-us/dotnet/api/overview/azure/messaging.servicebus-readme?view=azure-dotnet).
```

Comment:
Can you add some words here indicating that the `ApplicationProperties` dictionary is where we will add our DT headers?